### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by armp30**

### CODE REVIEW

File: repositories.txt [hunk 1]
```diff
@@ -4456,3 +4456,4 @@ https://github.com/dndubins/QuickStats
 https://github.com/vChavezB/uc-os3-arduino-due.git
 https://github.com/fabriziop/FIFOEE.git
 https://github.com/sparkfun/SparkFun_Qwiic_Fan_Arduino_Library
+https://github.com/viralinkio/ViraLink-MQTT-Client
```
Comment (armp30) [inline repositories.txt:4459]: fixed

**Comment by armp30** [comment]
@ArduinoBot

**Comment by armp30** [comment]
@ArduinoBot

**Comment by per1234** [comment]
Hello @armp30. I see you have fixed the error reported by the bot. However, Library Manager works exclusively from Git tags, so the bot won't be able to see the fix you made until you create a new [release](https://docs.github.com/en/github/administering-a-repository/releasing-projects-on-github/managing-releases-in-a-repository) or [tag](https://git-scm.com/docs/git-tag) in the library repository.

You can see here that the tagged version of the library being checked by the bot is still missing the `library.properties` file:

https://github.com/viralinkio/ViraLink-MQTT-Client/tree/0.50

Remember to update the `version` field in the library's [`library.properties` file](https://arduino.github.io/arduino-cli/latest/library-specification/#libraryproperties-file-format) as needed before making a release/tag.

Once you have done that, simply mention **@ArduinoBot** in a comment here to trigger it to run the automated checks again. After those are passing, your library will automatically be added to Library Manager.

**Comment by armp30** [comment]
@ArduinoBot

**Pull request merged**